source/extensions/tracers/well_known_names.h

```diff
@@ -21,6 +21,8 @@ class TracerNameValues {
   const std::string DynamicOt = "envoy.dynamic.ot";
   // Datadog tracer
   const std::string Datadog = "envoy.tracers.datadog";
+  // XRay tracer
+  const std::string XRay = "envoy.xray";
```

Comment:
The naming convention for tracers is `envoy.tracers.name`. Could you change this to `envoy.tracers.xray`?

Reply:
Sure thing. Will change it to `envoy.tracers.xray`.